src/vireshJJun20/Assignment20/ReverseNumber.java

```diff
@@ -0,0 +1,26 @@
+/*
+Program 3 WAP to reverse number (Consider Input as Integer not as String).
+Example-
+Input- 12345
+Output- 54321
+*/
+package vireshJJun20.Assignment20;
+
+public class ReverseNumber {
+
+	void reverseIntegers(int input) {
+		System.out.println("Input: " + input);
+		String output = "";
+		while (input > 0) {
+			int temp = input % 10;
+			output = output + String.valueOf(temp);
```

Comment:
Here you are printing output as a string and not as an integer .The logic needs to be changed.

Reply:
I went with the requirements. Only mentioned about the input data type. but I had updated the output data type per your suggestion. Thanks.

![image](https://user-images.githubusercontent.com/66830247/85954130-e5345f80-b992-11ea-8e52-d9252fef9165.png)

Reply:
Code is working as expected.approved.

Reply:
You can refer this. Deavina is correct. 
void findReverseNumber(int number) {
		int reverseNum = 0;
		int mod =0;
		while(number>0) {
			mod = number%10;
			number = number/10;
		    reverseNum = mod + reverseNum*10 ;
		}
		System.out.println("Reverse Number is "+reverseNum);
	}